### PR TITLE
Add a long DALI description in DALI wheel

### PR DESCRIPTION
**Pull request by JanuszL**
- adds a long DALI description in DALI wheel

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- adds a long DALI description in DALI wheel

### Affected modules and functionalities:
- setup.py.in for DALI and TF DALI plugin

### Key points relevant for the review:
- NA

### Tests:

- [x] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: dali_tf_plugin/setup.py.in [hunk 1]
```diff
@@ -36,6 +36,32 @@ class CustomDistribution(Distribution):
 
 setup(name='nvidia-dali-tf-plugin@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@',
       description='NVIDIA DALI @DALI_FLAVOR@ Tensorflow plugin for CUDA @CUDA_VERSION_SHORT@. Git SHA: @GIT_SHA@',
+      long_description='''Tensorflow plugin for NVIDIA DALI
+===========
+
+The Tensorflow plugin enabled usage of DALI with TensorFlow.
```
Comment (klecki) [inline dali_tf_plugin/setup.py.in:42]:
```suggestion
The TensorFlow plugin is required to use DALI with TensorFlow.
```

Comment (klecki) [reply dali_tf_plugin/setup.py.in:42]: Or 
```suggestion
The TensorFlow plugin enables usage of DALI with TensorFlow.
```

**Review by szalpal — APPROVED** [review]

File: dali_tf_plugin/setup.py.in (hunk 1 above)
Comment (JanuszL) [reply dali_tf_plugin/setup.py.in]: Done

**Review by klecki — APPROVED** [review]

File: dali_tf_plugin/setup.py.in [hunk 2]
```diff
@@ -36,6 +36,32 @@ class CustomDistribution(Distribution):
 
 setup(name='nvidia-dali-tf-plugin@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@',
       description='NVIDIA DALI @DALI_FLAVOR@ Tensorflow plugin for CUDA @CUDA_VERSION_SHORT@. Git SHA: @GIT_SHA@',
+      long_description='''Tensorflow plugin for NVIDIA DALI
```
Comment (klecki) [inline dali_tf_plugin/setup.py.in:39]:
```suggestion
      description='NVIDIA DALI @DALI_FLAVOR@ TensorFlow plugin for CUDA @CUDA_VERSION_SHORT@. Git SHA: @GIT_SHA@',
      long_description='''TensorFlow plugin for NVIDIA DALI
```

Comment (JanuszL) [reply dali_tf_plugin/setup.py.in]: Done

**Review by klecki — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Review by klecki — APPROVED** [review]

**Pull request merged**